pandas/core/arrays/interval.py

```diff
@@ -139,7 +139,6 @@
     )
 )
 class IntervalArray(IntervalMixin, ExtensionArray):
-    dtype = IntervalDtype()
```

Comment:
how is this working? do we already have this defined elsewhere?

Reply:
Exactly already defined as a class property